lib/aerospike/utils/unpacker.rb

```diff
@@ -85,7 +85,12 @@ def unpack_map(hash)
     def normalize_elem(elem)
       case elem
       when String
-       elem[1..-1].encode(Aerospike.encoding)
+        ptype = elem.ord
+        value = elem[1..-1]
+        if (ptype == ParticleType::STRING)
```

Comment:
I think this should be as below
```suggestion
        if (ptype == Aerospike::ParticleType::STRING)
```

Reply:
`Aerospike::Unpacker` and `Aerospike::ParticleType` are on the same level of the module hierarchy, so the `Aerospike::` namespace can be omitted.